pkgs/stdenv/darwin/make-bootstrap-tools.nix

```diff
@@ -132,15 +132,17 @@ in rec {
 
       cp -d ${lib.getLib darwin.libtapi}/lib/libtapi* $out/lib
 
-      ${lib.optionalString targetPlatform.isx86_64 ''
-        cp -rd ${pkgs.darwin.CF}/Library $out
+      cp -rd ${pkgs.darwin.CF}/Library $out
+      ${lib.optionalString stdenv.targetPlatform.isAarch64 ''
+        cp -rd ${pkgs.darwin.libobjc}/lib/* $out/lib/
```

Comment:
Both CoreFoundation and libobjc are tbds only on aarch64, but having these allows the CF test to pass.